docs/high_availability_installation_guide.md

```diff
@@ -15,7 +15,7 @@ Notices take these forms:
 ## Abstract 
 
 This guide describes how to install and configure Harbor for high availability. It supplements the [Installation Guides](installation_guide.md) and assumes that you are         familiar with the material in those guides.
-**Important**  This guide was last updated as of the Harbor 1.4.0 release. It will not apply to releases before 1.4.0. We advise that you read this at your own discretion when planning on your Harbor high availability implementation.  
+**Important**  This guide was last updated as of the Harbor 1.5.0 release. It will not apply to releases before 1.4.0. We advise that you read this at your own discretion when planning on your Harbor high availability implementation.  
```

Comment:
Should it be
```
... will not apply to releases before 1.5.0
```
???

Reply:
We support HA from 1.4, so it is 1.4 here.